doc/getting_started/setup_fpga.md

```diff
@@ -531,8 +531,8 @@ The FPGA tests attempt to load the latest bitstream by default, but because we w
 
 ```console
 # Load the bitstream with opentitantool
-./bazelisk.sh run //sw/host/opentitantool --interface=cw340 fpga load-bitstream <path_to_your_bitstream>
+./bazelisk.sh run //sw/host/opentitantool -- --interface=cw340 fpga load-bitstream <path_to_your_bitstream>
 
 # Run the broken test locally, showing all test output and skipping the bitstream loading
-./bazelisk.sh test <broken_test_rule> --define bitstream=skip --test_output=streamed
+./bazelisk.sh test <broken_test_rule> -- --define bitstream=skip --test_output=streamed
```

Comment:
I think this one is incorrect: `--define` and `--test_output` are bazel arguments so there is no need for `--`.